binding/web/template/src/rhino_worker.ts

```diff
@@ -25,7 +25,11 @@ let rhinoEngine: RhinoEngine = null;
 
 async function init(rhinoArgs: RhinoArgs): Promise<void> {
   try {
-    rhinoEngine = await Rhino.create(rhinoArgs.context);
+    rhinoEngine = await Rhino.create(
```

Comment:
so if you try the demo with empty or invalid accessKey in firefox or safari, you wont get any meaningful error, instead the console says something like `The object can not be cloned`, and i think the reason is that the `error` variable is a reference to an object that cannot be send as a message; one easy way to fix this is sending `error.toString()` instead of the original error.